extensions/amp-list/0.1/amp-list.js

```diff
@@ -825,6 +806,15 @@ export class AmpList extends AMP.BaseElement {
     return this.loadMoreFailedElement_;
   }
 
+  /** @private */
+  getLoadMoreEndElement_() {
+    if (!this.loadMoreEndElement_) {
+      this.loadMoreEndElement_ = childElementByAttr(
+          this.element, 'load-more-end');
```

Comment:
rules in `amp.css` needs to be updated for `amp-list[load-more-end]` as well.

I like to also make `amp.css` smaller, we should move the `amp-list[load-more] > [load-more-*].amp-visible` rules into an amp-list.css files as they can not be trigged until the extension has loaded anyway.